test/Serialization/PolynomialsSeries.jl

```diff
@@ -9,7 +9,7 @@ NonSimRel, c = number_field([y^2 - 5 * a, y^2 - 7 * a])
 Qu, u = rational_function_field(QQ, "u")
 Zt, t = polynomial_ring(residue_ring(ZZ, 2)[1], "t")
 Fin, d = Nemo.Native.finite_field(t^2 + t + 1)
-Frac = fraction_field(R)
+FracFieldElem = fraction_field(R)
```

Comment:
this and the change lower in the file are potentially dangerous. please change them to a variable name that's not used by type already